src/db.py

```diff
@@ -0,0 +1,34 @@
+__author__ = 'Tsintsir'
+
+from binascii import b2a_hex
+from os import urandom
+from pymongo import MongoClient
+from pbkdf2 import PBKDF2
+
+
+def create_user(username, email, password):
```

Comment:
I believe exceptions are appropriate for these cases, because they are exceptional (like, "you can't create that username, because that username already exists").

I don't think that failing to login deserves an exception, because it's not exceptional; when you call `create_user` there is no expectation that it should fail, but when you call `login` probably like 50% of the time you should expect failure. Although I do admit this is a somewhat arbitrary definition.